root/usr/bin/redsift/protocol.js

```diff
@@ -1,5 +1,14 @@
 'use strict';
 
+const convert = require('./convert.js');
+const fs = require('fs');
+
+const readFileContent = (file) => {
+  const ipc_root = process.env.IPC_ROOT;
+  const filename = ipc_root + '/' + file;
```

Comment:
better to use `path.join` here

Reply:
`${icp_root} / ${file}` ?

Reply:
```
const readFileContent = (file) =>
  fs.readFileSync(`${process.env.IPC_ROOT}/${file}`);
```

Reply:
I meant `filename = path.join(ipc_root, file)`